src/helpers/json_to_predreq.py

```diff
@@ -36,4 +36,5 @@ def decode(request):
         task=request.model["task"].lower(),
         featurizer=featurizers,
     )
+    dataset.select_features(SelectionList=request.model["selectedFeatures"])
```

Comment:
what if the selectedFeatures is an empty array?

Reply:
It is never empty. If not feature selection is done, then by default it contains all the features used.

Reply:
request.model["selectedFeatures"] returns an empty array for older models

Reply:
Maybe we need to change how it is implemented in jaqpotpyDataset class?

Reply:
I think it's easier to make it backwards compatible, by changing it to this:
```suggestion
    if (request.model['selectedFeatures'] && request.model['selectedFeatures'].length >0): 
        dataset.select_features(SelectionList=request.model["selectedFeatures"])
```

this will not break the previous uploaded models, and I don't think u need to change anything in jaqpotpy.